site/src/modules/dashboard/Navbar/UserDropdown/UserDropdownContent.tsx

```diff
@@ -161,15 +163,51 @@ export const UserDropdownContent: FC<UserDropdownContentProps> = ({
       <Divider css={{ marginBottom: "0 !important" }} />
 
       <Stack css={styles.info} spacing={0}>
-        <a
-          title="Browse Source Code"
-          css={[styles.footerText, styles.buildInfo]}
-          href={buildInfo?.external_url}
-          target="_blank"
-          rel="noreferrer"
-        >
-          {buildInfo?.version} <LaunchIcon />
-        </a>
+        <Tooltip title="Coder Version">
+          <a
+            title="Browse Source Code"
+            css={[styles.footerText, styles.buildInfo]}
+            href={buildInfo?.external_url}
+            target="_blank"
+            rel="noreferrer"
+          >
+            {buildInfo?.version} <LaunchIcon />
+          </a>
+        </Tooltip>
+
+        {buildInfo?.deployment_id && (
```

Comment:
Is there a chance that `deployment_id` could ever be an empty string?

I'm wondering if the `buildInfo?.deployment_id &&` could be swapped for `Boolean(buildInfo?.deployment_id) &&` to be a bit more explicit about disallowing any kind of falsey value

Reply:
Agreed!